kvbc/tools/sparse_merkle_db/include/sparse_merkle_db_editor.hpp

```diff
@@ -185,14 +185,33 @@ struct GetValue {
   }
 };
 
+struct RemoveMetadata {
+  std::string description() const {
+    return "removeMetadata\n"
+           "Removes metadata and state transfer data from RocksDB";
+  }
+  std::string execute(const v2MerkleTree::DBAdapter &db_adapter, const CommandArguments &) const {
+    const concordUtils::Sliver b{std::string{static_cast<char>(storage::v2MerkleTree::detail::EDBKeyType::BFT),
+                                             static_cast<char>(storage::v2MerkleTree::detail::EBFTSubtype::Metadata)}};
+    const concordUtils::Sliver e{std::string{static_cast<char>(storage::v2MerkleTree::detail::EDBKeyType::Key)}};
```

Comment:
I think we should add some checks to make sure we are safe in case the enum is changed. For example:

```
// Pseudocode

// Ensure Key is the next key type.
static_assert(EDBKeyType::BFT == EDBKeyType::Key + 1);

// Ensure Metadata is the first enum value.
static_assert(std::is_unsigned_v<EBFTSubtype>);
static_assert(EBFTSubtype::Metadata == 0);
```

Above won't solve the case of adding types to the `EBFTSubtype` enum after `Metadata` that we don't want to delete...

Another option is to not use `rangeDel()` and loop and only delete the types we want.

WDYT?

Reply:
I reviewed the solution and understood that we want to delete all the `BFT` keys.
Please take a look at the updated file.

Reply:
Ohk,, cool.